protocol/export_test.go

```diff
@@ -0,0 +1,9 @@
+package protocol
```

Comment:
In this PR it wouldn't be too hard, but this is preparation for other moving around that would make it much harder. (In short, `MemStore` will start depending directly on the `protocol` package, and [since it definitely wants to live in `prottest`] `protocol` can't also depend on `prottest`.)